MOBILE_CLIENT/COMMON/_HIGH/OVERLAY_VULNERABILITY/meta.json

```diff
@@ -0,0 +1,19 @@
+{
+    "risk_rating": "high",
+    "short_description": "Overlay Vulnerability is a type of vulnerability where an attacker can overlay malicious content on top of legitimate content, leading to potential security breaches.",
+    "references": {
+        "Overlay Attacks: Top Techniques And How To Counter Them": "https://www.appsealing.com/overlay-attacks/",
+        "Mobile overlay attacks on Android": "https://www.ikarussecurity.com/en/mobile-device-management-en/mobile-overlay-attacks-on-android/"
+    },
+    "title": "Overlay Vulnerability",
+    "privacy_issue": true,
+    "security_issue": true,
+    "categories": {
+        "OWASP_MASVS_L1": [
+            "MSTG_PLATFORM_2"
+        ],
+        "OWASP_MASVS_L2": [
+            "MSTG_PLATFORM_2"
+        ]
```

Comment:
I think these should can be added too: 

`"PCI_STANDARDS":[
      "REQ_2_2",
      "REQ_6_2",
"REQ_6_3"
    ]`